mesonbuild/backend/backends.py

```diff
@@ -19,7 +19,7 @@
 import typing as T
 import hashlib
 
-from .. import build
+from .. import build as _build
```

Comment:
Maybe a TODO comment here would be good?

Reply:
What did you have in mind?

Reply:
Why is it called `_build` here? It's not imported like this anywhere else.